src/Fixer/AbstractPhpUnitFixer.php

```diff
@@ -227,4 +228,50 @@ private function makeDocBlockMultiLineIfNeeded(DocBlock $doc, Tokens $tokens, in
 
         return $doc;
     }
+
+    /**
+     * @return iterable<array{
+     *     index: int,
+     *     loweredName: string,
+     *     openBraceIndex: int,
+     *     closeBraceIndex: int,
+     * }>
+     */
+    protected function getPreviousAssertCall(Tokens $tokens, int $startIndex, int $endIndex): iterable
+    {
+        $functionsAnalyzer = new FunctionsAnalyzer();
+
+        for ($index = $endIndex; $index > $startIndex; --$index) {
+            $index = $tokens->getPrevTokenOfKind($index, [[T_STRING]]);
+
+            if (null === $index) {
+                return;
+            }
+
+            // test if "assert" something call
+            $loweredContent = strtolower($tokens[$index]->getContent());
+
+            if (! str_starts_with($loweredContent, 'assert')) {
```

Comment:
method moved as-is to parent, abstract class, but please don't change CS - pls reapply `./php-cs-fixer fix`